src/Npgsql/BackendMessages/RowDescriptionMessage.cs

```diff
@@ -350,34 +350,39 @@ internal FieldDescription Clone()
         return field;
     }
 
-    internal void GetInfo(Type? type, ref PgConverterInfo lastConverterInfo)
+    internal void GetInfo(Type? type, ref PgConverterInfo lastConverterInfo, out bool asObject)
```

Comment:
nit: maybe align terminology here rather than using asObject for the parameter name and IsBoxingConverter on PgConverterInfo?

Reply:
Yeah maybe, asObject is a bit shorter. It's also the operation the caller is expected to use (Read vs ReadAsObject)

Reply:
So in light of this I'd say no https://github.com/npgsql/npgsql/pull/5290#discussion_r1344048206

They're different concepts, one is describing what the converter itself is typed as, while here we want to describe the action the caller should take.